src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -5155,6 +5201,12 @@ namespace Akka.Util.Internal
         public static System.Text.StringBuilder AppendJoin<T>(this System.Text.StringBuilder sb, string separator, System.Collections.Generic.IEnumerable<T> values) { }
         public static System.Text.StringBuilder AppendJoin<T>(this System.Text.StringBuilder sb, string separator, System.Collections.Generic.IEnumerable<T> values, System.Action<System.Text.StringBuilder, T, int> valueAppender) { }
     }
+    public class static TaskEx
```

Comment:
We have already had a file `src/core/Akka/Util/Internal/TaskExtensions.cs`, we could use it for those extensions